src/tersets.zig

```diff
@@ -150,3 +150,17 @@ pub fn getMaxMethodIndex() usize {
 
     return max_index;
 }
+
+/// `Point` is a point represented by `time` and `value` of datatype `time_type` and `value_type`.
+fn Point(comptime time_type: type, comptime value_type: type) type {
```

Comment:
Unclear why the value type is generic instead of always `f64` like everywhere else in the library.

Reply:
I did it generic to provide more flexibility but it is---indeed---so far, always f64.